mw-webapp/src/utils/getChunkArray.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Split array on subArrays
+ */
+export const getChunksArray = <T>(array: T[], chunkLength: number): T[][] => {
+  const chunksArray = [];
```

Comment:
it looks here like chunks array has type any[], but it is T[][]

Reply:
Added type